src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -311,14 +358,14 @@ private void WorkerLoop()
 
                         // We have one or more active operations. Run any work that needs to be run.
                         CURLMcode performResult;
-                        while ((performResult = Interop.Http.MultiPerform(multiHandle)) == CURLMcode.CURLM_CALL_MULTI_PERFORM);
+                        while ((performResult = Interop.Http.MultiPerform(_multiHandle)) == CURLMcode.CURLM_CALL_MULTI_PERFORM);
```

Comment:
I'd assert that _multiHandle isn't null, and isn't IsInvalid; since this can be on a recycled instance.